.circleci/src/workflows/release.yml

```diff
@@ -215,6 +225,7 @@ jobs:
   - commit-audius-docker-compose-and-notify:
       context: [slack-secrets, github]
       requires:
+        - push-core
```

Comment:
we don't want this blocked on this workflow right?

Reply:
at least not yet?

Reply:
tagging the images will fail if we don't first ensure it's been pushed. The build should remain stable even as we figure out the tests.